project/Dependencies.scala

```diff
@@ -27,6 +27,7 @@ object Dependencies {
   val pureConfig = "com.github.pureconfig" %% "pureconfig" % "0.12.2"
   val scalaTest = "org.scalatest" %% "scalatest" % "3.1.0"
   val scalazzi = "com.github.vovapolu" %% "scaluzzi" % "0.1.3"
+  val sortImports = "com.nequissimus" %% "sort-imports" % "0.3.2"
```

Comment:
Does it work reliably? Last time I tried it it removed some imports that were essential

Reply:
has been working reliably for me so far :man_shrugging: